scripts/generate-test-services.ts

```diff
@@ -97,3 +95,76 @@ async function generateAll(): Promise<void> {
 }
 
 generateAll();
+
+function removeImports(str: string): string {
```

Comment:
I guess this line and below are the adjustment you mentioned to avoid touching the generator.

Reply:
Yes, Imports are removed and replace by the ones from common which are sufficient for a entity with links. This will fail if we change the generation for entity strongly i.e. add different fields etc. But we will see this in a failling test so I guess this is ok.